modules/build/controllers/stack/stack.go

```diff
@@ -222,16 +221,6 @@ func populateRbac(stack *riov1.Stack, saName, systemNamespace, buildKitPodName s
 		},
 	}
 
-	role2 := rbac.NewRole(stack.Namespace, fmt.Sprintf("%s-stack", stack.Name), nil)
-	role2.Rules = []v1.PolicyRule{
-		{
-			APIGroups:     []string{"rio.cattle.io"},
-			Resources:     []string{"stacks"},
-			ResourceNames: []string{stack.Name},
-			Verbs:         []string{"get", "update"},
-		},
-	}
```

Comment:
This wasn’t used apparently ?

Reply:
Yeah, it is using rio-standard user right now.